feature-find-meal/src/main/java/com/kinandcarta/feature/find/meal/adapter/MealsAdapter.kt

```diff
@@ -39,6 +39,11 @@ class MealsAdapter(
         notifyDataSetChanged()
     }
 
+    fun updateMeal(position: Int, meal: Meal) {
+        mealsList[position] = meal
+        notifyItemChanged(position)
```

Comment:
Could you not update just the quantity here, as that is what has changed, and then you wouldn't need the distance code for the reserve call?

Reply:
That would do the trick and mean nothing else needs to change on API side.  Well spotted!